integrationtest/mapper/pojo-base/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexedEmbeddedBaseIT.java

```diff
@@ -1617,10 +1822,682 @@ class EntityC {
+	public void parentExcludeChildIncludeMultipleResultingInEmbeddedBeingIncludedPartially() {
+		class Model {
+			@Indexed(index = INDEX_NAME)
+			class IndexedEntity {
+				@DocumentId
+				Integer id;
+				@KeywordField
+				String indexedEntityString;
+
+				@IndexedEmbedded(excludePaths = "subIncluded.subSubIncluded.subSubExcludedString")
+				Included included;
+
+				public IndexedEntity(Integer id, String indexedEntityString, Included included) {
+					this.id = id;
+					this.indexedEntityString = indexedEntityString;
+					this.included = included;
+				}
+			}
+
+			class Included {
+				@KeywordField
+				String includedString;
+
+				@IndexedEmbedded
+				SubIncluded subIncluded;
+
+				public Included(String includedString, SubIncluded subIncluded) {
+					this.includedString = includedString;
+					this.subIncluded = subIncluded;
+				}
+			}
+
+			class SubIncluded {
+				@KeywordField
+				String subIncludedString;
+
+				@IndexedEmbedded(includePaths = { "subSubIncludedString", "subSubExcludedString" })
+				SubSubIncluded subSubIncluded;
+
+				public SubIncluded(String subIncludedString, SubSubIncluded subSubIncluded) {
+					this.subIncludedString = subIncludedString;
+					this.subSubIncluded = subSubIncluded;
+				}
+			}
+
+			class SubSubIncluded {
+				@KeywordField
+				String subSubIncludedString;
+				@KeywordField
+				String subSubExcludedString;
+
+				public SubSubIncluded(String subSubIncludedString) {
+					this.subSubIncludedString = subSubIncludedString;
+					this.subSubExcludedString = subSubIncludedString;
+				}
+			}
+		}
+
+		backendMock.expectSchema( INDEX_NAME, b -> b
+				.field( "indexedEntityString", String.class )
+				.objectField( "included", b2 -> b2
+						.field( "includedString", String.class )
+						.objectField( "subIncluded", b3 -> b3
+								.field( "subIncludedString", String.class )
+								.objectField( "subSubIncluded", b4 -> b4
+										.field( "subSubIncludedString", String.class )
+								)
+						)
+				)
+		);
+
+		SearchMapping mapping = setupHelper.start()
+				.withAnnotatedEntityTypes( Model.IndexedEntity.class )
+				.setup();
+
+		backendMock.verifyExpectationsMet();
+
+		Model model = new Model();
+
+		doTestEmbeddedRuntime(
+				mapping,
+				id -> model.new IndexedEntity(
+						id, "a",
+						model.new Included( "b", model.new SubIncluded(
+								"c",
+								model.new SubSubIncluded( "d" )
+						) )
+				),
+				document -> document.field( "indexedEntityString", "a" )
+						.objectField( "included", b2 -> b2
+								.field( "includedString", "b" )
+								.objectField( "subIncluded", b3 -> b3
+										.field( "subIncludedString", "c" )
+										.objectField( "subSubIncluded", b4 -> b4
+												.field( "subSubIncludedString", "d" )
+										)
+								)
+						)
+		);
+	}
+
+	/*
+	 * Since parent adds a path to be excluded and child uses an include filter that explicitly includes a different field -- parent won't find a filed it tries to exclude
```

Comment:
```suggestion
	 * Since parent adds a path to be excluded and child uses an include filter that explicitly includes a different field -- parent won't find a field it tries to exclude
```